ocaml/xapi/xapi_vm_migrate.ml

```diff
@@ -821,20 +820,40 @@ let migrate_send'  ~__context ~vm ~dest ~live ~vdi_map ~vif_map ~options =
   eject_cds __context cd_vbds;
 
   try
+    (* Sort VDIs by size in principle and then age secondly. This gives better
+       chances that similar but smaller VDIs would arrive comparatively
+       earlier, which can serve as base for incremental copying the larger
+       ones. *)
+    let compare_fun v1 v2 =
```

Comment:
In theory it could be named ``compare``. Naming it differently is just to avoid confusion with the pervasive ``compare`` in ``Pervasives``. Also it uses ``compare`` inside, so might be misleading to let people think it's a recursive definition.